M2/Macaulay2/m2/hypertext.m2

```diff
@@ -4,6 +4,8 @@
 needs "regex.m2" -- for toLower
 needs "lists.m2" -- for all
 needs "methods.m2"
+needs "packages.m2" -- for Package
+needs "monoids.m2" -- for Monoid etc
```

Comment:
AHHH no this is very bad! `hypertext.m2` is near the top of `loadsequence`, `packages.m2` and `monoids.m2` are far, far below. The few hypertext definitions should move there instead.

Reply:
This was fine in `html.m2` because it's near the bottom, so admittedly this is a consequence of my own suggestion, but fortunately this files doesn't seem to use Monoid at all and `hypertext Package` can safely move to `packages.m2`.

Reply:
hehe I told you I had a bad feeling about moving things around...